MC/bin/o2dpg_sim_workflow.py

```diff
@@ -976,13 +998,36 @@ def getDigiTaskName(det):
      workflow['stages'].append(tpcclus)
      tpcreconeeds.append(tpcclus['name'])
 
+   # ===| TPC reco |===
+   tpcLocalCFreco=dict()
+
+   # handle distortion corrections and scaling using MC maps
+   # this assumes the lumi inside the maps is stored in FT0 (pp) scalers
+   # in case of PbPb the conversion factor ZDC ->FT0 (pp) must be set
+   tpc_corr_options_mc=''
+
+   if tpcDistortionType == 0: # disable distortion corrections
+      tpc_corr_options_mc=' --corrmap-lumi-mode 0 '
+      tpcLocalCFreco['TPCCorrMap.lumiMean'] = '-1';
+   elif tpcDistortionType == 1: # disable scaling
+      tpc_corr_options_mc=' --corrmap-lumi-mode 2 '
+      tpcLocalCFreco['TPCCorrMap.lumiInst'] = str(CTPSCALER)
+      tpcLocalCFreco['TPCCorrMap.lumiMean'] = str(CTPSCALER)
+   elif tpcDistortionType == 2: # full scaling with CTP values
+      if COLTYPE == 'PbPb':
+         tpcLocalCFreco['TPCCorrMap.lumiInstFactor'] = str(lumiInstFactor)
+      tpc_corr_options_mc=' --corrmap-lumi-mode 2 '
+      tpcLocalCFreco['TPCCorrMap.lumiInst'] = str(CTPSCALER)
```

Comment:
This does not correspond to what was done for digitization.
Here, `TPCCorrMap.lumiInst` and `TPCCorrMap.lumiInstFactor` are set separately whereas for digitization `TPCCorrMap.lumiInst` is set directly to the factor.

Is it because the configKey values are digested differently in reconstruction and digitization?

Reply:
Indeed, the digizer does not use the lumiInstFactor https://github.com/AliceO2Group/AliceO2/blob/0129938292bf379849ec58652be5c1cfa88e60f1/Detectors/TPC/simulation/src/Digitizer.cxx#L227
While to correction map in reco does: https://github.com/AliceO2Group/AliceO2/blob/0129938292bf379849ec58652be5c1cfa88e60f1/Detectors/TPC/calibration/src/CorrectionMapsLoader.cxx#L201
For reconstruction I wanted to mirror what is done in data. But I assume one could also directly multiply here, right @shahor02 ?